### PR TITLE
PHP SDK 2.x updates

### LINKED ISSUE #153: Update PHP tutorials to use Microsoft Graph SDK 2.x

### PR DESCRIPTION
**Pull request by jasonjoh**
Fixes #153

### CODE REVIEW

File: tutorials/includes/php/06-get-inbox-messages.md [hunk 1]
```diff
@@ -68,14 +68,14 @@ Unlike the `getUser` function from the previous section, which returns a single
...
 
 > [!NOTE]
-> The value passed in `setPageSize` is an upper-bound, not an explicit number. The API returns a number of messages *up to* the specified value.
+> The value passed in `queryParameters->top` is an upper-bound, not an explicit number. The API returns a number of messages *up to* the specified value.
 
 #### Getting subsequent pages
 
-If there are more results available on the server, collection responses include an `@odata.nextLink` property with an API URL to access the next page. The PHP SDK exposes this as the `isEnd` method on collection request objects. If this method returns false, there are more results available. The next page can be accessed by the `getPage` method.
+If there are more results available on the server, collection responses include an `@odata.nextLink` property with an API URL to access the next page. The PHP SDK exposes this as the `getOdataNextLink` method on collection request objects. If this method returns a non-empty string, there are more results available.
```
Comment (Ndiritu) [inline tutorials/includes/php/06-get-inbox-messages.md:78]: Should we link to the "Paging" section of the "Use SDK" docs here for a developer to know how to get the next page(s) of data? Using the raw nextlink URL is not as straight-forward using v2.

File: tutorials/includes/php/04-add-user-auth.md [hunk 2]
```diff
@@ -6,6 +6,14 @@ ms.localizationpriority: medium
...
+
+The Microsoft Graph SDK includes authentication providers based on the [PHP League OAuth2 client](https://github.com/thephpleague/oauth2-client). However for this tutorial, you will use the [device code flow](/azure/active-directory/develop/v2-oauth2-device-code) to obtain access tokens. The included authentication providers do not implement this flow, so you will implement a custom access token provider.
+
+1. Create a new file in the root directory of your project named **DeviceCodeTokenProvider.php**. Add the following code.
+
+    :::code language="php" source="./src/user-auth/graphtutorial/DeviceCodeTokenProvider.php":::
+
 ## Configure Graph client for user authentication
 
 In this section you will use the `GuzzleHttp\Client` class to request an access token by using the [device code flow](/azure/active-directory/develop/v2-oauth2-device-code).
```
[the remark this answers is not in the document]
Comment (Ndiritu) [reply tutorials/includes/php/04-add-user-auth.md:19]: Same for the App-only auth tutorial

**Review by Ndiritu — COMMENTED** [review]
Explanation section of sending mail might need an update as well: [Creating Objects](https://learn.microsoft.com/en-us/graph/tutorials/php?tabs=aad&tutorial-step=6)

File: tutorials/includes/php/06-get-inbox-messages.md (hunk 1 above)
Comment (jasonjoh) [reply tutorials/includes/php/06-get-inbox-messages.md:78]: Only issue is there are no PHP snippets on that page :D.

Comment (Ndiritu) [reply tutorials/includes/php/06-get-inbox-messages.md:78]: Hopefully once https://github.com/microsoftgraph/microsoft-graph-docs/pull/20527 is merged :)

**Review by Ndiritu — APPROVED** [review]
Thanks for this @jasonjoh!

**Pull request merged**